test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionProtocolTests.cs

```diff
@@ -266,7 +293,57 @@ public class HubConnectionProtocolTests
                 await connection.ReceiveJsonMessage(new { invocationId = "1", type = 2, item = 42 }).OrTimeout();
 
                 var ex = await Assert.ThrowsAsync<InvalidOperationException>(() => invokeTask).OrTimeout();
-                Assert.Equal("Streaming methods must be invoked using HubConnection.Stream", ex.Message);
+                Assert.Equal("Streaming methods must be invoked using StreamAsync.", ex.Message);
+            }
+            finally
+            {
+                await hubConnection.DisposeAsync().OrTimeout();
+                await connection.DisposeAsync().OrTimeout();
+            }
+        }
+
+        [Fact]
```

Comment:
Note to myself - need to add a counterpart of this for the Stream method as well as a test case for invoking a streaming method returning an empty stream with Invoke